### ISSUE TITLE
PostgreSQL: Data type Money not supported

### ISSUE THREAD

**Issue by kaelte** [serge-rider/dbeaver#1217]
Columns of type `money` are displayed empty (`null`) and warning appears:
```
create table public.price (name text,price money) ;
insert into public.price (name,price) values ('A',10);

select * from public.price ;
```
The result set appears as
```
name |price |
-----|------|
A    |      |
```
together with the following warning:
```
SQL Error [22003]: Unzulässiger Wert für den Typ double : Fr. 10,00.

org.jkiss.dbeaver.model.exec.DBCException: SQL Error [22003]: Unzulässiger Wert für den Typ double : Fr. 10,00.
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:56)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetDataReceiver.fetchRow(ResultSetDataReceiver.java:107)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.fetchQueryData(SQLQueryJob.java:603)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.executeSingleQuery(SQLQueryJob.java:356)
	at org.jkiss.dbeaver.runtime.sql.SQLQueryJob.extractData(SQLQueryJob.java:708)
	at org.jkiss.dbeaver.ui.editors.sql.SQLEditor$QueryResultsContainer.readData(SQLEditor.java:1686)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetJobDataRead.run(ResultSetJobDataRead.java:92)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:103)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: org.postgresql.util.PSQLException: Unzulässiger Wert für den Typ double : Fr. 10,00.
	at org.postgresql.jdbc.PgResultSet.toDouble(PgResultSet.java:2922)
	at org.postgresql.jdbc.PgResultSet.getDouble(PgResultSet.java:2294)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetImpl.getDouble(JDBCResultSetImpl.java:396)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCNumberValueHandler.fetchColumnValue(JDBCNumberValueHandler.java:140)
	at org.jkiss.dbeaver.model.impl.jdbc.data.handlers.JDBCAbstractValueHandler.fetchValueObject(JDBCAbstractValueHandler.java:50)
	... 8 more
```

In the command line I can see the price:
```
$ psql -c 'select * from public.price;'
 name |   price   
------+-----------
 A    | Fr. 10,00
(1 row)
```

**Comment by serge-rider** [comment]
This is PostgreSQL JDBC driver issue (https://github.com/pgjdbc/pgjdbc/issues/100). I can't fix it.

Workaround: 
```sql
select price::numeric from public.price ;
```
(I knows that this is a bad workaround but I don't know better solution for now).

**Comment by greenpeas** [comment]
Sergei, it seems to me with the driver all right. Recheck it please.
![image](https://user-images.githubusercontent.com/3856344/29510126-61a6ae8a-8663-11e7-84de-dc21146006e5.png)

**Comment by serge-rider** [comment]
It is actually works in DBeaver too. At least with latest drivers and PG.

**Comment by serge-rider** [comment]
Resolved in #1573 (it became possible with recent fixes in PG driver)
